plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryWithDifferentProjectIdConnectorTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.QueryRunner;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestBigQueryWithDifferentProjectIdConnectorTest
+        extends BaseBigQueryConnectorTest
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        return BigQueryQueryRunner.createQueryRunner(
+                ImmutableMap.of(),
+                ImmutableMap.of("bigquery.project-id", getProjectId().get()),
+                REQUIRED_TPCH_TABLES);
+    }
+
+    @Override
+    protected Optional<String> getProjectId()
+    {
+        return Optional.of("bigquery-cicd-alternate");
```

Comment:
Can you update the README in trino-bigquery to mention that a 2nd project needs to exist and MUST be named bigquery-cicd-alternate?

IMO we should make the name configurable like we do with `-Dtesting.gcp-storage-bucket="trino-ci-test"`.